javascript/javascript2/week1/problem-solving-cardio.js

```diff
@@ -0,0 +1,45 @@
+//Find the shortest word
+
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+
+function getShortestWord() {
+  let shortestWord = danishWords[0];
+  for (let i = 1; i < danishWords.length; i++) {
+    if (danishWords[i].length < danishWords[i + 1].length) {
```

Comment:
your code all good , 
but for quick code break- put 'Ø' as last element in the array then see what happens.
IT's broken right ? this is because danishWords[i+1], which is not existed after the last element in the array.

Try replacing danishWords[i+1] with shortestWord.length. still error? 
see comment below.